include/exiv2/basicio.hpp

```diff
@@ -292,6 +292,7 @@ class EXIV2API FileIo : public BasicIo {
     @param path The full path of a file
    */
   explicit FileIo(const std::string& path);
+  explicit FileIo(const std::wstring& path);
```

Comment:
I'm curious if this should be guarded by _WIN32 as well.

Reply:
Yes it should be guarded as well. I'll add #idef's...
Or alternativelly it can be converted to internal UTF-8 name.